core/controllers/topics_and_skills_dashboard.py

```diff
@@ -176,15 +175,15 @@ def post(self):
         """Handles POST requests."""
 
         classroom_name = self.payload.get('classroom_name')
-        num_queries_to_fetch = int(self.payload.get('items_per_page'))
+        num_items_to_fetch = int(self.payload.get('num_items_to_fetch'))
```

Comment:
Sorry, just realized "items" is vague. Please change to "skills" instead.

Reply:
Yeah sorry, I also should have brainstormed a bit. 
Done.